### PR TITLE
Add note about configuration docs assuming framework defaults are loaded

### PR DESCRIPTION
**Pull request by flanger001**
From https://github.com/rails/rails/pull/35928 @gmcgibbon

### CODE REVIEW

File: guides/source/configuring.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ config.active_record.schema_format = :ruby
 
 Rails will use that particular setting to configure Active Record.
 
+The default values listed in the next section all assume you have the framework defaults loaded for your version of Rails.
+In a Rails 6.0 app, for example, this is set as `config.load_defaults(6.0)` in `config/application.rb`.
```
Comment (gmcgibbon) [inline guides/source/configuring.md]: The defaults mentioned would be specific to the version of the guide (eg. https://guides.rubyonrails.org/v5.2/, https://edgeguides.rubyonrails.org/). I wonder if we can insert the version into the document with erb...? 🤔

Comment (flanger001) [reply guides/source/configuring.md]: Is there a mechanism for that somewhere? I get the concern about the version but I wanted to include a version.

Comment (gmcgibbon) [reply guides/source/configuring.md]: So we can't insert the version into the document. The documentation itself is versioned.

New plan: I now see there is a "Results of `load_defaults`" header further down the document, and an entry for the `form_with_generates_ids` config option already. I'll re-open your old PR and merge that change.

Sorry for the confusion.

Comment (flanger001) [reply guides/source/configuring.md]: Hey @gmcgibbon, it's all good, thank you for merging anyway!

**Pull request closed without merging**